cartography/intel/aws/ec2/vpc_peerings.py

```diff
@@ -0,0 +1,173 @@
+import logging
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_vpc_peerings(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    return client.describe_vpc_peering_connections()['VpcPeeringConnections']
+
+
+@timeit
+def load_vpc_peerings(neo4j_session, data, region, aws_account_id, aws_update_tag):
+
+    ingest_vpc_peerings = """
+    UNWIND {vpc_peerings} AS vpc_peering
+
+    MERGE (pcx:PeeringConnection:AWSPeeringConnection{id: vpc_peering.VpcPeeringConnectionId})
+    ON CREATE SET pcx.firstseen = timestamp()
+    SET pcx.lastupdated = {aws_update_tag},
+    pcx.allow_dns_resolution_from_remote_vpc =
+        vpc_peering.RequesterVpcInfo.PeeringOptions.AllowDnsResolutionFromRemoteVpc,
+    pcx.allow_egress_from_local_classic_link_to_remote_vpc =
+        vpc_peering.RequesterVpcInfo.PeeringOptions.AllowEgressFromLocalClassicLinkToRemoteVpc,
+    pcx.allow_egress_from_local_vpc_to_remote_classic_link =
+        vpc_peering.RequesterVpcInfo.PeeringOptions.AllowEgressFromLocalVpcToRemoteClassicLink,
+    pcx.requester_region = vpc_peering.RequesterVpcInfo.Region,
+    pcx.accepter_region = vpc_peering.AccepterVpcInfo.Region,
+    pcx.status_code = vpc_peering.Status.Code,
+    pcx.status_message = vpc_peering.Status.Message
+
+    MERGE (avpc:AWSVpc{id: vpc_peering.AccepterVpcInfo.VpcId})
+    ON CREATE SET avpc.firstseen = timestamp()
+    SET avpc.lastupdated = {aws_update_tag}, avpc.vpcid = vpc_peering.AccepterVpcInfo.VpcId
+
+    MERGE (rvpc:AWSVpc{id: vpc_peering.RequesterVpcInfo.VpcId})
+    ON CREATE SET rvpc.firstseen = timestamp()
+    SET rvpc.lastupdated = {aws_update_tag}, rvpc.vpcid = vpc_peering.RequesterVpcInfo.VpcId
+
+    MERGE (aaccount:AWSAccount{id: vpc_peering.AccepterVpcInfo.OwnerId})
+    ON CREATE SET aaccount.firstseen = timestamp(), aaccount.foreign = true
```

Comment:
I don't think this treatment of `foreign` is the same as the older vpc_peering.py. In that file, I believe `foreign` was used to denote AWS accounts that were not included in the set of accounts that we "initially knew about", if that makes sense. We can talk about this in chat too.

If you get stuck on what to do here, I think it's fine to not set `foreign` for now.